sqldb-postgres/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "wasmcloud-provider-sqldb-postgres"
```

Comment:
By convention this should just be wasmcloud-provider-postgres but it's fine this way.